modules/apps/blogs/blogs-editor-configuration/src/main/java/com/liferay/blogs/editor/configuration/internal/BlogsCoverImageCaptionEditorConfigContributor.java

```diff
@@ -55,13 +56,23 @@ public void populateConfigJSONObject(
 		String removePlugins = jsonObject.getString("removePlugins");
 
 		if (Validator.isNotNull(removePlugins)) {
-			removePlugins = removePlugins + ",magicline";
+			removePlugins = removePlugins + ",magicline,stylescombo,videoembed";
 		}
 		else {
 			removePlugins = "magicline";
 		}
 
-		jsonObject.put("removePlugins", removePlugins);
+		jsonObject.put(
+			"removePlugins", removePlugins
+		).put(
+			"toolbarImage", ""
+		).put(
+			"toolbarTable", ""
+		).put(
+			"toolbarText", "TextLink"
```

Comment:
Doesn't this override text toolbar with just one button? That can't be right.

Reply:
It was necessary to achieve the "only-button" toolbar

See the current master:

<img width="365" alt="Screen Shot 2021-08-30 at 12 35 43" src="https://user-images.githubusercontent.com/7663513/131365147-6cb62b75-a990-4d0b-b4f0-97308c01d821.png">

Reply:
That looks so weird :smile: . But, if their product folk like it, :man_shrugging: